internal/printer/json2table/jsonpaths/apigateway.go

```diff
@@ -0,0 +1,36 @@
+package jsonpaths
+
+var (
+	ApiGatewayGateway = map[string]string{
+		"Id":             "id",
+		"Name":           "properties.name",
+		"Logs":           "properties.logs",
+		"Metrics":        "properties.metrics",
+		"Status":         "metadata.status",
+		"PublicEndpoint": "metadata.publicEndpoint",
```

Comment:
add StatusMessage like you did on Route, it is important to see the reason why the resource reached a certain state